src/Reader/ReadableDataInterface.php

```diff
@@ -36,7 +21,8 @@ public function withLimit(int $limit): static;
     public function read(): iterable;
 
     /**
-     * Get a next item from the data set.
+     * Get the first item from the data set.
```

Comment:
In this variant the method seems to be useless and wrong-named. Better name would be `readFirst()` but I'm not sure why is it needed.

Initial idea was:

```php
while ($row = $dataReader->readOne()) {
  // do something
}

// instead of

foreach ($dataReader->read() as $row) {
   // do something with
}
```

but now I see that it doesn't add much and could be removed.

Reply:
Actually, it might make sense if the meaning is "read one item" but the how is up to implementation. Use cases are: pick random item, highlight first item, highlight last item etc.

```suggestion
     * Get an item from the data set.
```